### PR TITLE
add swagger for wishlists/{id}

### PR DESCRIPTION
**Pull request by yf1357**
image

### CODE REVIEW

File: tests/test_service.py [hunk 1]
```diff
@@ -227,9 +227,6 @@ def test_get_wishlist_not_found(self):
         resp = self.app.get("/wishlists/0")
         self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
         data = resp.get_json()
-        self.assertEqual(data['error'], "Not Found")
-        self.assertEqual(data['message'], ("404 Not Found:"
```
Comment (abhinavGupta16) [inline tests/test_service.py:231]: We can edit the message to be checked rather than removing it.

File: tests/test_service.py [hunk 2]
```diff
@@ -571,9 +568,6 @@ def test_update_wishlist_with_unsupported_media_type(self):
                          status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
         data = resp.get_json()
         app_type = "application/json"
-        self.assertEqual(data["message"],
```
Comment (abhinavGupta16) [inline tests/test_service.py:574]: We can edit the message to be checked rather than removing it.

**Review by abhinavGupta16 — CHANGES_REQUESTED** [review]

File: tests/test_service.py (hunk 1 above)
Comment (yf1357) [reply tests/test_service.py:231]: Done.

**Review by abhinavGupta16 — APPROVED** [review]

**Review by sqz0914 — APPROVED** [review]

**Comment by abhinavGupta16** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b7dd79b7-17cc-412e-98f1-e1a234aafd1c?env_id=ibm:yp:us-south "nyu-wishlist-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/76e35b85-cc07-4faf-b3ae-ee3dbe88d89f "nyu-wishlist-service-f20-pipeline") deployed [wishlists](https://github.com/wishlists/wishlists/releases/tag/deploy-ibm-yp-us-south-ag7387-nyu.edu-prod-20201125-033337 "deploy-ibm-yp-us-south-ag7387-nyu.edu-prod-20201125-033337") to [prod](https://console.bluemix.net/apps/b9e11a39-3222-4ca4-a89b-8f6a16a809d0?env_id=ibm:yp:us-south "ibm:yp:us-south:ag7387@nyu.edu:prod"), including commit 043ae26abfe338b3914353b0b583548f78e42098

**Comment by abhinavGupta16** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b7dd79b7-17cc-412e-98f1-e1a234aafd1c?env_id=ibm:yp:us-south "nyu-wishlist-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/76e35b85-cc07-4faf-b3ae-ee3dbe88d89f "nyu-wishlist-service-f20-pipeline") deployed [wishlists](https://github.com/wishlists/wishlists/releases/tag/deploy-ibm-yp-us-south-ag7387-nyu.edu-dev-20201128-205903 "deploy-ibm-yp-us-south-ag7387-nyu.edu-dev-20201128-205903") to [dev](https://console.bluemix.net/apps/87d9b0cd-cdb0-4349-b181-099124ba4b73?env_id=ibm:yp:us-south "ibm:yp:us-south:ag7387@nyu.edu:dev"), including commit 043ae26abfe338b3914353b0b583548f78e42098

**Pull request merged**